src/coordination/azure/Akka.Coordination.Azure/AzureLease.cs

```diff
@@ -92,12 +97,18 @@ public override async Task<bool> Release()
                 if(_log.IsDebugEnabled)
                     _log.Debug("Releasing lease");
                 var result = await _leaseActor.Ask(LeaseActor.Release.Instance, _timeout);
-                return result switch
+                switch (result)
                 {
-                    LeaseActor.LeaseReleased _ => true,
-                    LeaseActor.InvalidRequest req => throw new LeaseException(req.Reason),
-                    _ => throw new LeaseException($"Unexpected response type: {result.GetType()}")
-                };
+                        case LeaseActor.LeaseReleased:
+                            return true;
+                        case LeaseActor.InvalidReleaseRequest:
+                            _log.Info("Tried to release a lease that is not acquired");
```

Comment:
I'd mark this as `DEBUG`

Reply:
it needs to be `INFO` because if its not, we could not debug if a lease was "released" because it was actually released or because we didn't have it in the first place